cost_test.go

```diff
@@ -127,7 +128,7 @@ func TestCost(t *testing.T) {
 			schema: genMapSchema(nil, withRule(genStringSchema(nil), `self == self`)),
 			expectedErrors: []*CostError{
 				{
-					Path: "<root>.<properties>",
+					Path: field.NewPath("openAPIV3Schema", "<properties>"),
```

Comment:
How would the path k/k returns for an error for the same field look?

Reply:
Right, the path set here did not reflect the k/k path (which would look instead something like `spec.validation.openAPIV3Schema.additionalProperties`) - this should be fixed now.